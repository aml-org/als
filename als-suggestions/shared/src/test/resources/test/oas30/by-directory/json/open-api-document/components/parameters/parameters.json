{
  "openapi": "3.0",
  "components": {
    "parameters": {
      "test": {
        *
      }
    }
  }
}