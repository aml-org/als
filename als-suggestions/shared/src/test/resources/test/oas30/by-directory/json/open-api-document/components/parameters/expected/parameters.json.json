[
  {
    "label": "$ref",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "$ref: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "description",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "description: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "name",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "name: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "in",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "in: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "required",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "required: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "deprecated",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "deprecated: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "allowEmptyValue",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "allowEmptyValue: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "style",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "style: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "explode",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "explode: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "allowReserved",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "allowReserved: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "schema",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "schema: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "example",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "example: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "examples",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "examples: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  },
  {
    "label": "content",
    "kind": [],
    "detail": "",
    "documentation": "",
    "deprecated": [],
    "preselect": [],
    "sortText": "",
    "filterText": null,
    "insertText": null,
    "insertTextFormat": [
      1
    ],
    "textEdit": {
      "range": {
        "start": {
          "line": x,
          "character": x
        },
        "end": {
          "line": x,
          "character": x
        }
      },
      "newText": "content: "
    },
    "additionalTextEdits": null,
    "commitCharacters": null,
    "command": null
  }
]